ci-operator/step-registry/baremetalds/assisted/operator/lso/create-disks/baremetalds-assisted-operator-lso-create-disks-ref.yaml

```diff
@@ -1,6 +1,6 @@
 ref:
   as: baremetalds-assisted-operator-lso-create-disks
-  from: dev-scripts
+  from: src
```

Comment:
Shouldn't it be assisted-service? So for example when we would like to trigger this job from other repositories.